### PR TITLE
Week 2 meeting

### PR DESCRIPTION
**Pull request by jrtechs**
Drafted week 2 meeting email.

-- Don't merge -- will merge after campus group email is sent.

### CODE REVIEW

File: announcements/_posts/2019-09-06-w02-everthing-ssh.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: "Week #2: Everything SSH!"
+layout: post-event
+date-start: "2019-09-06 16:30"
+date-end: "2019-09-06 18:00"
+location: "GOL-2620 (Med. DB Lab)"
+---
+
+Hey Ritluggers,
+
+This week we got a very excitting meeting: we are diving deep into SSH.
```
Comment (Tjzabel) [inline announcements/_posts/2019-09-06-w02-everthing-ssh.md]: * First phrase before the colon is phrased awkwardly. Either "This week we have...", or another variation. "This week we got..." isn't valid English in this context :P
* excitting(sp).

**Comment by jrtechs** [comment]
lol, I just merged in from master and tried pushing and wondered why it wasn't working. @Tjzabel thanks!

File: announcements/_posts/2019-09-06-w02-everthing-ssh.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+We are going to cover everything from basic SSH commands to some of the more advance topics.
```
Comment (Tjzabel) [inline announcements/_posts/2019-09-06-w02-everthing-ssh.md]: "...of the more **advanced**..."

File: announcements/_posts/2019-09-06-w02-everthing-ssh.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+You need no background knowledge of SSH for this meeting, just show up and learn with us.
+
+After a quick presentation there will be a SSH CTF (capture the flag) esk challenge.
```
Comment (Tjzabel) [inline announcements/_posts/2019-09-06-w02-everthing-ssh.md]: The *esk* isn't the correct usage. I know what you're trying to say here, but I don't think that's the proper usage in a writing context.

**Comment by Tjzabel** [comment]
@jrtechs your branch was out of date with master. I pulled the latest changes in. I'm also not sure if codeclimate is still running due to that, or if it's still being pulled in somewhere.

**Comment by jrtechs** [comment]
@Tjzabel a few things were funky with that push. First I branched from the presentation branch so I didn't get the squashed commits from the master. Second, I didn't save the file before pushing so it didn't get all of my last second spell check changes :laughing: 

I'll just squash when merging into master.

File: announcements/_posts/2019-09-06-w02-everthing-ssh.md (hunk 3 above)
Comment (jrtechs) [reply announcements/_posts/2019-09-06-w02-everthing-ssh.md]: how about:
After a quick presentation, there will be a SSH CTF-esk (capture the flag) challenge.

**Pull request merged**